Feature crud customer list

Thraxelon opened the pull request:
Added multiple file for CRUD Customer process
- Index.php : modified to show CRUD Function link
- view_customerCRUD.php : File for all CRUD Activity
> In this, the data shown is only CID, Customername, 1st line of Address, and current Customer Status
- CRUD-CreateCustomer.php : Create Customer activity
- CRUD-DetailsCustomer.php : Read Details Customer activity
- CRUD-UpdateCustomer.php : Update Customer activity
- CRUD-DeleteCustomer.php : Delete Customer activity
- CRUD-functions.php : All global CRUD functions are here
- customers.inc.php : Class for Customer Object.
Attachment: Display for CRUD Activity page (view_customerCRUD.php)
![image](https://user-images.githubusercontent.com/58166832/87777435-aa1ba280-c853-11ea-83e4-d9f2bb57bc1f.png)

Commit:
modified UI display

Commit:
add functions for CRUD activity file

Commit:
added main View page for CRUD activity

Commit:
added CUSTOMER class for CRUD activity

Commit:
added page for new Customer Creation activity

Commit:
added page to show Customer Details

Commit:
added page to update Customer

Commit:
Changed namespace to CRUD\Customer\Create

Commit:
Changed namespace to CRUD\Customer\Update
Set all textbox into readonly

Commit:
Commented out debug console

Commit:
added Update function

Commit:
updated Edit button, redirects to Update page

Commit:
added Delete customer function

Commit:
added Delete function in class

Commit:
Enables delete button in main View
Adds javascript for dialog popup before deletion.
Adds Info label on top of page when delete function is done

Commit:
update Public infront of functions

michaelmeganet reviewed (changes requested):
The customer must input their mobile phone no. , you have to add a phone no. field

echo "<a href=\"javascript:delValidate('CRUD-DeleteCustomer.php?cid={$customerData['cid']}','#')\" class='btn btn-danger'>Delete</a>";
List the source of the above javascript:delValidate in a commented message to brief your intend to apply it here , in view_customerCRUD.php

![view_customerCRUD](https://user-images.githubusercontent.com/5449215/87791297-9c2a4980-c874-11ea-9962-f7e67f300647.png)

Commit:
add main selection for CRUD process

Commit:
adds validation for Create and Update funciton

Commit:
Changed CRUD file location into CRUD folder

Commit:
added a header for CRUD functions, contains jquery validation

Commit:
added phone number, moved to CRUD folder

Commit:
commented out debug texts

Commit:
changed link for CRUD into CRUD-index.php

Commit:
Merged from upstream : customer.inc.php

Commit:
updated customers table, added phone column;

Thraxelon commented:
I've updated the customer table to have phone number,
Added validation method for CRUD create and CRUD update.

Changed file location all in one folder, CRUD.
Main acccess to this files is using CRUD-index.php, from index.php
